### PR TITLE
Feature/hg18 unittest parsing html from wap

### PR DESCRIPTION
**Pull request by Sheena309**

### CODE REVIEW

**Review by bdfinlayson — CHANGES_REQUESTED** [review]
@Sheena309 once you've resolved comments on feature/HG21-add-progress-bars please pull those changes into this branch, since it looks like you've branched off of that one to create this one

File: tests/headline_grabber/pipeline_steps/test_scrape_text_wap.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+        site_configs=[config],
+        headlines=[],
+        grouped_headlines={},
+        documents_for_display={},
+        user_input=UserPreferences(
+            include=['wap'],
+            exclude=None,
+        ),
+    )
+    actualResult = scrape_text.run(pipeline_context)
```
Comment (bdfinlayson) [inline tests/headline_grabber/pipeline_steps/test_scrape_text_wap.py:147]: you shouldn't be running the entire step unless you plan to mock the html requests, which you're not. Either mock the html requests, or pass in a stub html string to the get headlines beautifulsoup method

**Review by bdfinlayson — CHANGES_REQUESTED** [review]

**Comment by Sheena309** [comment]
> @Sheena309 once you've resolved comments on feature/HG21-add-progress-bars please pull those changes into this branch, since it looks like you've branched off of that one to create this one

Done!

**Review by bdfinlayson — APPROVED** [review]

**Review by bdfinlayson — APPROVED** [review]

**Review by bdfinlayson — APPROVED** [review]

**Review by bdfinlayson — APPROVED** [review]
LGTM

**Review by bdfinlayson — DISMISSED** [review]

**Review by tonthattuit — APPROVED** [review]
Looks good to me. Approved!

**Review by tonthattuit — APPROVED** [review]

**Pull request merged**